packages/next/taskfile.js

```diff
@@ -354,23 +351,24 @@ export async function ncc_edge_runtime_cookies() {
   // `@edge-runtime/cookies` is precompiled and pre-bundled
   // so we vendor the package as it is.
   const dest = 'src/compiled/@edge-runtime/cookies'
-  const pkg = await fs.readJson(
+  const pkg = await readJson(
     require.resolve('@edge-runtime/cookies/package.json')
   )
-  await fs.remove(dest)
+  await rmrf(dest)
+  await fs.mkdir(dest, { recursive: true })
 
-  await fs.outputJson(join(dest, 'package.json'), {
+  await writeJson(join(dest, 'package.json'), {
     name: '@edge-runtime/cookies',
     version: pkg.version,
     main: './index.js',
     license: pkg.license,
   })
 
-  await fs.copy(
+  await fs.cp(
     require.resolve('@edge-runtime/cookies/dist/index.js'),
     join(dest, 'index.js')
   )
-  await fs.copy(
+  await fs.cp(
     require.resolve('@edge-runtime/cookies/dist/index.d.ts'),
     join(dest, 'index.d.ts')
   )
```

Comment:
I think these need the option `{ force: true }`

Reply:
The dest directory is removed and recreated just before these.